src/commands/push.ts

```diff
@@ -95,24 +96,29 @@ async function importHandler(path: string, params: ImportParams) {
   const result = await prepareImport(client, files);
   if (params.forceMode === 'NO' && hasConflicts(result)) {
     error(
-      "There are conflicts. Resolve them in the browser or set --forceMode option to 'KEEP' or 'OVERRIDE'."
+      "There are conflicts. Resolve them in the browser or set --force-mode option to 'KEEP' or 'OVERRIDE'."
```

Comment:
We should print a link to the platform where they can resolve it. Also the `ask` function would help here.

Reply:
So we can even open the browser on the platform.